package.json

```diff
@@ -35,47 +35,47 @@
     "aws-sdk": "^2.306.0",
     "bcryptjs": "latest",
     "body-parser": "^1.15.2",
-    "chai": "^3.5.0",
-    "chai-http": "^3.0.0",
+    "chai": "^4.2.0",
+    "chai-http": "^4.2.0",
     "chance": "^1.0.12",
     "client-oauth2": "^4.1.0",
     "compression": "^1.6.2",
     "console-stamp": "^0.2.5",
     "cors": "^2.8.1",
     "crypto-js": "^3.1.9-1",
     "ejs": "^2.5.5",
-    "email-templates": "^2.5.4",
-    "es6-promisify": "^5.0.0",
+    "email-templates": "^5.0.2",
+    "es6-promisify": "^6.0.1",
     "express": "^4.14.0",
     "feathers": "^2.0.3",
     "feathers-authentication": "^1.0.2",
     "feathers-authentication-jwt": "^0.3.1",
-    "feathers-authentication-local": "^0.3.2",
-    "feathers-configuration": "^0.3.3",
+    "feathers-authentication-local": "^0.4.4",
+    "feathers-configuration": "^0.4.2",
     "feathers-errors": "^2.5.0",
-    "feathers-hooks": "^1.7.1",
-    "feathers-mongoose": "^3.6.2",
+    "feathers-hooks": "^2.1.2",
+    "feathers-mongoose": "^6.2.0",
     "feathers-rest": "^1.6.0",
     "feathers-socketio": "^2.0.1",
-    "feathers-swagger": "^0.3.2",
-    "file-type": "^9.0.0",
+    "feathers-swagger": "^0.7.2",
+    "file-type": "^10.5.0",
     "freeport": "^1.0.5",
     "jwt-decode": "^2.2.0",
     "lodash": "^4.17.4",
```

Comment:
Wird in den global hooks verwendet, ist aber inzwischen überholt, da alles wofür es verwendet wurde mit nodejs vanila abbilden lässt.